metron-platform/metron-elasticsearch/src/main/java/org/apache/metron/elasticsearch/dao/ElasticsearchMetaAlertDao.java

```diff
@@ -720,4 +719,12 @@ public int getPageSize() {
   public void setPageSize(int pageSize) {
     this.pageSize = pageSize;
   }
+
+  private String getField(String globalConfigKey, String defaultField) {
```

Comment:
It seems like we have duplicated this logic here and in `SearchServiceImpl`.  Would it make sense to extract this out?

Reply:
The latest commit extracts this to ConfigurationsUtils class in metron-common.  This class is already really big so I was hesitant to add it there but I'm not sure where else it belongs.